FlexiblePopupPanels/Panel.js

```diff
@@ -208,7 +209,7 @@ const PanelBase = kind({
 						/>
 					</Cell>
 					<Cell
-						className={css.content}
+						className={classnames(css.content, (typeof ENACT_PACK_NO_ANIMATION !== 'undefined' && ENACT_PACK_NO_ANIMATION) ? css.noAnimation : null)}
```

Comment:
Could we change this to use `computed` like I mentioned in `Checkbox`?

Reply:
ok
I changed `FlexiblePopupPanels/Header.js` file and less file too.